### DISCUSSION TITLE
Charting options (.to_chart()) and indicators like RSI

### DISCUSSION THREAD

**Discussion by caballerofelipe** [OpenBB-finance/OpenBB#6069] [Q&A]
I'm using the to_chart() method, which works fine, I'm wondering if there''s documentation on how to use it, like the parameters that can be used. For instance I want my data to show RSI.

So I guess I want the following things:
- Documentation on how to use to_chart()
- Know how to get stuff like RSI (and others, BBands, SMA...) for my data and add it to a Dataframe (without computing it manually).
- Plot those things on a chart (the easy way).

**Comment by IgorWounds** [comment]
@deeleeramone @hjoaquim

**Comment by hjoaquim** [comment]
The place to look into atm is the PyPi page of the charting extension:
https://pypi.org/project/openbb-charting/
which provides useful insight on how to use it.

Also, one can look into the method's doc strings directly:
![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/48914296/b91d195c-c1c9-4ffc-bc93-cc4e1f126066)
